internal/payloads/error_payload.go

```diff
@@ -28,6 +28,13 @@ type ResponseError struct {
 	Context context.Context `json:"-"`
 }
 
+type ClientResponseError struct {
+	ResponseError
+
+	// Response Body from Client
+	ClientErrors []byte `json:"-"`
```

Comment:
Why plural? From what I can see, the client returns one error. It can probably contain wrapped error, but in that case the generic consensus in Go is to simply use singular.

Is there a particular reason why you do not want this to be marshaled to JSON? I mean, it can be pretty useful for API users to actually see the root cause.

Reply:
https://github.com/RHEnVision/provisioning-backend/blob/70009fc58bfed0a287266135c9a9eac955678968/internal/clients/sources/client.gen.go#L624
I am getting from sources an array of errors and I am parsing all of them.